test/functional/increment-unserialized-model.js

```diff
@@ -0,0 +1,65 @@
+import { assert } from 'chai';
+import { addAction } from '../util/database';
+import { patch } from '../util/mock-client';
+import server from '../../.dist/lib/server';
+import ModelStore from '../../.dist/store/model-store';
+
+describe('Edit model', () => {
+    let model;
+
+    before(() => {
+        return server.start(8080);
+    });
+
+    after(() => {
+        return server.stop();
+    });
+
+    beforeEach(() => {
+        return addAction('CLEAR_ALL_DATA').then(() => {
+            return addAction('NEW_MODEL', {
+                name: 'Resistor',
+                description: 'V = IR',
+                manufacturer: 'Pancakes R\' Us',
+                vendor: 'Mouzer',
+                location: 'Shelf 14',
+                allowCheckout: true,
+                price: 10.50,
+                count: 20
+            }).then(actionId => {
+                model = ModelStore.getModelByActionId(actionId);
+            });
+        });
+    });
+
+    it('should increment a model', () => {
+        let newModel = {
```

Comment:
Nit: It think it'd be better if it was just `let newModel = model; newModel.count = 21; newModel.instock = 21;`